src/ucp/core/ucp_mm.c

```diff
@@ -377,17 +359,60 @@ static ucs_status_t ucp_memh_register(ucp_context_h context, ucp_mem_h memh,
     return UCS_OK;
 }
 
+static ucs_status_t
+ucp_memh_create_no_rcache(ucp_context_h context, void *address,
```

Comment:
maybe
ucp_memh_create
since sometimes it's used to create user_memh, even if there is an rcache (from ucp_memh_alloc for example)